addon/src/helpers/page-title.ts

```diff
@@ -10,26 +16,28 @@ import { guidFor } from '@ember/object/internals';
  */
 export default class PageTitle extends Helper {
   @service('page-title')
-  tokens;
+  declare tokens: PageTitleService;
 
-  get tokenId() {
+  get tokenId(): string {
     return guidFor(this);
   }
 
   constructor() {
+    // eslint-disable-next-line prefer-rest-params
     super(...arguments);
     this.tokens.push({ id: this.tokenId });
   }
 
-  compute(params, _hash) {
-    let hash = {
+  compute(params: string[], _hash: PageTitleHelperOptions) {
```

Comment:
Then this will type check, though you'll want to be explicit about it returning `void`:

```suggestion
  compute(params: string[], _hash: PageTitleHelperOptions): void {
```